autobuild/generate.py

```diff
@@ -1,59 +1,57 @@
 import os
+from os import path
 import glob
 import shutil
+import sys
+import yaml
 
-import generate_util
 import build_util
+import generate_autofit
 
+project = sys.argv[1]
 
 BUILD_PATH = os.getcwd()
-WORKSPACE_PATH = f"{os.getcwd()}/../autofit_workspace"
+WORKSPACE_PATH = f"{os.getcwd()}/../{project}_workspace"
 SCRIPTS_ROOT_PATH = f"{WORKSPACE_PATH}/scripts"
 NOTEBOOKS_ROOT_PATH = f"{WORKSPACE_PATH}/notebooks"
+CONFIG_PATH = f"{BUILD_PATH}/autobuild/config"
 
-COPY_VEBATIM_FILES = [
-    "./howtofit/chapter_1_introduction/gaussian.py",
-    "./howtofit/chapter_1_introduction/profiles.py",
-    "./howtofit/chapter_database/profiles.py",
-]
-NOTEBOOKS_REMOVE = [
-    "./howtofit/chapter_1_introduction/gaussian.ipynb",
-    "./howtofit/chapter_1_introduction/profiles.ipynb",
-    "./howtofit/chapter_database/profiles.ipynb",
-    "./overview/new_model_component/linear_fit.ipynb",
-]
+with open(path.join(CONFIG_PATH, "copy_files.yaml"), "r+") as f:
+    copy_files_dict = yaml.load(f)
 
-def main():
+copy_files_list = copy_files_dict[project]
 
-    generate_util.generate_project_folders()
+with open(path.join(CONFIG_PATH, "notebooks_remove.yaml"), "r+") as f:
+    notebooks_remove_dict = yaml.load(f)
+
+notebooks_remove_list = notebooks_remove_dict[project]
+
+
+if __name__ == "__main__":
+
+    if project == "autofit":
+        generate_autofit.generate_project_folders()
 
     os.chdir(SCRIPTS_ROOT_PATH)
 
     for x in [t[0] for t in os.walk(".")]:
 
-        if "__pycache__" in x:
-            try:
-                shutil.rmtree(x)
-            except FileNotFoundError:
-                pass
-
         scripts_path = f"{SCRIPTS_ROOT_PATH}/{x}"
         notebooks_path = f"{NOTEBOOKS_ROOT_PATH}/{x}"
-
+        if "__pycache__" in x:

```

Comment:
I'm surprised there's __pycache__ directories present. That should only occur for local testing unless they got committed by mistake?